src/utils/parser.js

```diff
@@ -144,6 +145,22 @@ function isVariable(path) {
     (t.isObjectProperty(path) && !isFunction(path.node.value));
 }
 
+function getMemberExpression(root) {
+  function _getMemberExpression(node, expr) {
+    if (node.type === "MemberExpression") {
```

Comment:
Rather than looking at the type directly with string comparison we could use `babel-type` helpers https://github.com/babel/babel/tree/master/packages/babel-types#memberexpression There are additional ones that match JSX member expressions and more so maybe a simple helper that does multiple checks like my `isVariable` or `isFunction` helpers.